src/Components/test/testassets/Ignitor/BlazorClient.cs

```diff
@@ -293,21 +294,11 @@ public async Task<bool> ConnectAsync(Uri uri, bool prerendered, bool connectAuto
                 return true;
             }
 
-            // Now everything is registered so we can start the circuit.
-            if (prerendered)
-            {
-                CircuitId = await GetPrerenderedCircuitIdAsync(uri);
-                var result = false;
-                await ExpectRenderBatch(async () => result = await HubConnection.InvokeAsync<bool>("ConnectCircuit", CircuitId));
-                return result;
-            }
-            else
-            {
-                await ExpectRenderBatch(
-                    async () => CircuitId = await HubConnection.InvokeAsync<string>("StartCircuit", uri, uri),
-                    TimeSpan.FromSeconds(10));
-                return CircuitId != null;
-            }
+            var descriptor = await GetPrerenderDescriptors(uri);
+            await ExpectRenderBatch(
+                async () => CircuitId = await HubConnection.InvokeAsync<string>("StartCircuit", uri, uri, descriptor),
+                TimeSpan.FromSeconds(10));
```

Comment:
Is there something configurable we can use? Doesn't this class have a property for timeouts? Or just not pass one here and it behaves reasonably?

Reply:
We used to default to 500ms which wasn't enough apparently for StartCircuit to succeed

Reply:
Added a property specifically to control this